keanu-project/src/main/java/io/improbable/keanu/vertices/Vertex.java

```diff
@@ -18,62 +18,29 @@
     private boolean observed;
 
     /**
-     * This is the value of the probability density at the supplied value.
+     * This is the value of the natural log probability at the supplied value. In the
```

Comment:
is it natural log prob or natural log of the prob?

Reply:
Changed to `This is the natural log of the probability at the supplied value.`